AccountMultithreading/AccountMultithreading.java

```diff
@@ -0,0 +1,105 @@
+class Account {
```

Comment:
It is not recommended to create multiple classes in one java file.

Reply:
Resolved